deploy-disconnected-registry/olm-sync.sh

```diff
@@ -66,12 +72,19 @@ function mirror() {
         check_registry ${DESTINATION_REGISTRY}
     fi
 
-    echo ">>>> Podman Login into Source Registry: ${SOURCE_REGISTRY}"
-    ${PODMAN_LOGIN_CMD} ${SOURCE_REGISTRY} -u ${REG_US} -p ${REG_PASS} --authfile=${PULL_SECRET}
-    ${PODMAN_LOGIN_CMD} ${SOURCE_REGISTRY} -u ${REG_US} -p ${REG_PASS}
-    echo ">>>> Podman Login into Destination Registry: ${DESTINATION_REGISTRY}"
-    ${PODMAN_LOGIN_CMD} ${DESTINATION_REGISTRY} -u ${REG_US} -p ${REG_PASS} --authfile=${PULL_SECRET}
-    ${PODMAN_LOGIN_CMD} ${DESTINATION_REGISTRY} -u ${REG_US} -p ${REG_PASS}
+
+    if [[ ${CUSTOM_REGISTRY} == "false" ]]; then
+        echo ">>>> Podman Login into Source Registry: ${SOURCE_REGISTRY}"
+        ${PODMAN_LOGIN_CMD} ${SOURCE_REGISTRY} -u ${REG_US} -p ${REG_PASS} --authfile=${PULL_SECRET}
+        ${PODMAN_LOGIN_CMD} ${SOURCE_REGISTRY} -u ${REG_US} -p ${REG_PASS}
+        echo ">>>> Podman Login into Destination Registry: ${DESTINATION_REGISTRY}"
+        ${PODMAN_LOGIN_CMD} ${DESTINATION_REGISTRY} -u ${REG_US} -p ${REG_PASS} --authfile=${PULL_SECRET}
+        ${PODMAN_LOGIN_CMD} ${DESTINATION_REGISTRY} -u ${REG_US} -p ${REG_PASS}
+    else
```

Comment:
Just to ensure: We did 2 times before (in the if block) because we need to get login (the first time) and then, we need to do a merge using the second flag --authfile.
In your case, is not needed because you've got directly using the authfile with creds right?

Reply:
correct